client/state/sites/hooks/use-site-global-styles-status.ts

```diff
@@ -62,19 +46,16 @@ const getGlobalStylesInfoForSite = ( siteId: number | null ): Promise< GlobalSty
 		return Promise.resolve( {
 			shouldLimitGlobalStyles: true,
 			globalStylesInUse: false,
-			globalStylesInPersonalPlan: false,
+			globalStylesInPersonalPlan: true,
 		} );
 	}
 
 	if ( siteId === null ) {
-		return loadExperimentAssignment( 'calypso_global_styles_personal_v2' ).then(
-			( experimentAssignment ) =>
-				Promise.resolve( {
-					shouldLimitGlobalStyles: true,
-					globalStylesInUse: false,
-					globalStylesInPersonalPlan: experimentAssignment.variationName === 'treatment',
-				} )
-		);
+		return Promise.resolve( {
+			shouldLimitGlobalStyles: true,
+			globalStylesInUse: false,
+			globalStylesInPersonalPlan: true,
```

Comment:
You can remove this property.

Reply:
[As above](https://github.com/Automattic/wp-calypso/pull/79683#discussion_r1269649306)